.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Primary owner should be listed first in list of global owners, followed by any secondary owners
-*       @SirSpidey @ocofaigh
+*       @rajatagarwal-ibm @ocofaigh
```

Comment:
@rajatagarwal-ibm please remove me as code owner - perhaps @toddgiguere might like to be secondary owner since he has some experience with schematics agents?

Reply:
I'm good with that.

Reply:
OK